packages/clay-date-picker/src/Helpers.ts

```diff
@@ -17,6 +19,36 @@ export type WeekDays = Array<IDay>;
 
 export type Month = Array<WeekDays>;
 
+export const RANGE_SEPARATOR = ' - ';
+
+export function isDateRangeWithinYears(year: number, years: IYears) {
+	return year >= years.start && years.end >= year;
```

Comment:
While this works, it doesn't read quite as naturally as it could. This is subtle, but let me see if I can explain what I mean.

Say you have a range of years that starts at `A` and ends at `Z`, and you want to find out whether year `J` falls somewhere in the middle. ie. on a timeline, is J in the middle:

```
- - - - A - - - - - - - - - - J - - - - - - Z - - - - →
      (start year)    (candidate year)   (end year)
```

It's possible to write you function so that the `true` result happens (ie. the "happy" path), the variables corresponding to `A`, `J` and `Z` appear in your expression that order:

```
//         A           J       J         Z
return years.start <= year && year <= years.end;
```

(In your version, the order is `J` `A`, `Z`, `J`)

Anyway, very subjective, but food for thought.

Reply:
Makes sense for me!!